### DISCUSSION TITLE
Is it possible to set a notification sound according to a notification importance level?

### DISCUSSION THREAD

**Discussion by danimesq** [phuhl/linux_notification_center#211] [Q&A]
For example, dunst does support notification importance level:

<a href="https://gist.github.com/fwSmit/9127d988b07bcec9d869f2c927d0f616">
<img alt="screenshot2" src="contrib/screenshots/screenshot2_cut.png">
</a>

**Comment by phuhl** [comment]
Hi @danimesq, currently sound is not supported at all, see here: https://github.com/phuhl/linux_notification_center/issues/82. Any sound you might be hearing when using the deadd notification center is produced by the client application that send the notification.

But I am open to PRs that add sound capabilities :)

**Comment by danimesq** [comment]
This feature would be quite useful.

Cinnamon DE supports setting a custom sound when playing notifications, and I would love if linux_notification_center adds this feature

**Reply by phuhl** [reply]
Sure, the feature is very welcome. I am not currently available to build it, though. If you want, you can get your feet wet and try to implement it. In the above mentioned issue are some hints on how the implementation could work. If you have any questions while building such a feature, feel free to ask.
